app/src/main/cpp/skyline/services/serviceman.h

```diff
@@ -61,8 +58,13 @@ namespace skyline::service {
          * @note This only works for services created with `NewService` as sub-interfaces used with `RegisterService` can have multiple instances
          */
         template<typename Type>
-        inline std::shared_ptr<Type> GetService(Service serviceType) {
-            return std::static_pointer_cast<Type>(serviceMap.at(serviceType));
+        std::shared_ptr<Type> GetService(ServiceName name) {
+            return std::static_pointer_cast<Type>(serviceMap.at(name));
+        }
+
+        template<typename Type>
+        constexpr std::shared_ptr<Type> GetService(std::string_view name) {
```

Comment:
const ref, a `std::string_view` is still 2 64-bit ints long and shouldn't be copied where possible

Reply:
The compiler will do this itself, same as std span